TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -34,9 +36,9 @@ class TransformationCleaningAgent( AgentModule ):
   """
   .. class:: TransformationCleaningAgent
 
-  :param DataManger dm: DataManager instance
-  :param TransfromationClient transClient: TransfromationClient instance
-  :param FileCatalogClient metadataClient: FileCatalogClient instance
+  :param ~DIRAC.DataManagementSystem.Client.DataManager.DataManager dm: DataManager instance
+  :param ~TransformationClient.TransformationClient transClient: TransformationClient instance
+  :param ~FileCatalogClient.FileCatalogClient metadataClient: FileCatalogClient instance
```

Comment:
If it is just FileCatalogClient sphinx complains about multiple targets for it because Module and class have the same name. If I use `~FileCatalogClient.FileCatalogClient` the only target is the class. The tilde makes it only print the last part of the dotted path. DataManager is included somewhere else so the full path is given

http://diracsailer.readthedocs.io/en/fixdoc/CodeDocumentation/TransformationSystem/Agent/TransformationCleaningAgent.html